README.md

```diff
@@ -42,7 +41,7 @@ module "this" {
 
 | Name | Version |
 |------|---------|
-| <a name="provider_aws"></a> [aws](#provider\_aws) | >= 4.0.0 |
+| <a name="provider_aws"></a> [aws](#provider\_aws) | 4.58.0 |
```

Comment:
Has there perhaps been an uncommitted change to the provider to introduce this change to the provider version?   This should be reverted as we can't force the latest minor version I don't think.

Reply:
I think this should just be ">= 4.0.0" ?

Is the auto doc generator blasting it away?